README.md

```diff
@@ -17,8 +17,8 @@ instance, in an object recognition network, an attribution method could tell us
 which pixels of the image were responsible for a certain label being picked, or
 which words from sentence were indicative of strong sentiment.
 
-Applications range from helping a developer debug, allowing analysts to explore
-the logic of a network, and to give end-user’s some transparency into the reason
+Applications range from helping a developer debug over allowing analysts to explore
```

Comment:
To make this easier to read, can we change to:
Applications include helping developers debug their models, allowing analysts to explore the logic of a model, and giving end-users some transparency into the reason for a network's prediction.

Reply:
yes, that is much better, changed accordingly